functional-tests/src/test/java/com/redhat/che/functional/tests/BayesianAbstractTestClass.java

```diff
@@ -98,6 +98,7 @@ public void closeFiles() {
     closeFilesAndProject();
   }
 
+  @Override
```

Comment:
It doesn't make sense for me to override that. If there is a problem with the method with same name, maybe changing the name would be enough?

Reply:
I got a warning that the method is overriding a method from `super`, which it is, and that the `@Override` annotation was missing, so I added it.

Reply:
Basically the idea is to add @AfterClass annotation to the method. I'm not sure if it is needed to @Override it as far as the body of the method is exactly the same as in the parent. 
Wouldn't it be better to use something like that?
```
  @AfterClass
  public void closeBrowserAfterTests() {	  
    super.closeBrowser();	    
  }	  
```

Reply:
if it's exactly the same as parent, then maybe the implementation of the `@AfterClass` in the test case is not needed. the `@AfterClass` of parent class should be called, right?

Reply:
But there is no @AfterClass in a parent. And as far as parent should be common for all test cases, I'm not sure if we should set methods like @BeforClass and @AfterClass there. I think only the behaviour should be implemented and exact class inheriting from that should say if it wants to use that method in specific way (e.g. as @AfterClass).